setup.py

```diff
@@ -64,5 +84,4 @@
         'Topic :: Software Development :: Libraries',
         'Topic :: Scientific/Engineering',
         'Topic :: Scientific/Engineering :: Bio-Informatics'],
-    platforms=['any'],
```

Comment:
in the "classifier" section, please listed supported OSes: Windows, Mac OS X, (specific linux distros?)